getting-started-with-codespaces.md

```diff
@@ -0,0 +1,30 @@
+# Using Codespaces to work with the "Practical Deep Learning for Coders" course
+
+
+To get started, create a codespace for this repository by clicking this 👇 
+
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?hide_repo_select=true&ref=master&repo=485606685)
+
+A codespace will open in a web-based version of Visual Studio Code.
+
+**Note**: Dev container spec is also supported by [other supporting tools](https://containers.dev/supporting) along with GitHub Codespaces.
```

Comment:
nit: Maybe explain a bit that dev containers are an open spec (so we don't assume users know this), and then mention the spec is supported by Codespaces and other tools?

Reply:
Updated!